email_signup.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"text/template"
+
+	mail "github.com/xhit/go-simple-mail/v2"
+)
+
+var (
+	successMessageTemplate = "Dear {{.}},\n Thank you for registering with us. We are glad to have you on board. \n Regards, \n HNG Team"
+	t                      = createTemplate("t2", successMessageTemplate)
+)
+
+func sendEmail(email string, message string) error {
+	server := mail.NewSMTPClient()
+	server.Host = "smtp.mailtrap.io"   // change this to take variable from config
+	server.Port = 2525                 // change this to take variable from config
+	server.Username = "f59d2513ff6024" // change this to take variable from config
+	server.Password = "8f448ea24f5f0e" // change this to take variable from config
+	server.Encryption = mail.EncryptionTLS
```

Comment:
who is gonna change it?
load this variables from a .env file
store this example values is a sample.env file
these files should be at the root directory